### PR TITLE
Add my pin to the map

### PR DESCRIPTION
**Pull request by atikahmd**
Adding pin

### CODE REVIEW

File: atikahmd.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+gitHandle: atikahmd
```
Comment (crichID) [inline atikahmd.json]: @atikahmd this should be `githubHandle` instead of `gitHandle`

**Comment by crichID** [comment]
@atikahmd :wave: you are very close. There are just a couple of small things to fix so your tests will pass:

- It should say `githubHandle` instead of `gitHandle`
- You will need to place your file inside the `_pins` directory.

Instructions for fixing the file can be found in this video: https://vimeo.com/201011288

Let us know if you need any help!

**Comment by NickiCB** [comment]
@atikahmd Looks good

**Comment by baruchMo** [comment]
@atikahmd. wouderful

**Comment by brianamarie** [comment]
@atikahmd I went ahead and fixed the syntax issues in [571704f](https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/5655/commits/571704fad3b6101b78040f0cee633e0247708e24). Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 28, I will go ahead and do it just to keep the repository clean!

**Pull request merged**